be/src/vec/sink/mysql_result_writer.cpp

```diff
@@ -87,6 +87,11 @@ Status MysqlResultWriter::_add_one_column(const ColumnPtr& column_ptr) {
             }
         }
 
+        if constexpr (type == TYPE_BOOLEAN) {
+            //todo here need to using uint after MysqlRowBuffer support it
+            buf_ret = _vec_buffers[i]->push_tinyint(static_cast<int8_t>(column->get_int(i)));
```

Comment:
I suggest use assert_cast<const ColumnVector<Uint8>&>(*column).get_data() instead of  column->get_int.

Reply:
good suggestion